.github/workflows/security-scan.yml

```
@@ -0,0 +1,87 @@
+      alerts_medium: ${{ steps.report.outputs.alerts_medium }}
+      alerts_high: ${{ steps.report.outputs.alerts_high }}
+    steps:
+    - uses: actions/checkout@v4
+
+    - name: Setup dependencies
+      run: |
+        npm install
+
+    - name: Run static security scan
+      run: |
+        npm audit --json > npm-audit-report.json 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)